src/ui/components/TransferGrain.js

```diff
@@ -18,102 +18,192 @@ export type Props = {|
   +setLedger: (Ledger) => void,
 |};
 
+const useStyles = makeStyles({
+  root: {
+    width: "1100px",
+    margin: "0 auto",
+    background: "white",
+    padding: "0 5em 5em",
+  },
+  arrowBody: {
+    width: "300px",
+    marginLeft: "20px",
+    background: "#C9CED2",
+    padding: "5px 20px",
+  },
+  triangle: {
+    width: 0,
+    height: 0,
+    background: "white",
+    borderTop: "30px solid transparent",
+    borderBottom: "30px solid transparent",
+    borderLeft: "30px solid #C9CED2",
+  },
+  dropdown: {
+    width: "300px",
+    height: "50px",
+    border: "1px solid #9EADBA",
+    borderRadius: "500px",
+    background: "#F2F5F7",
+  },
+  centerRow: {
+    display: "flex",
+    justifyContent: "center",
+    alignItems: "center",
+  },
+  third: {
+    width: "300px",
+    height: "50px",
+    margin: "30px 15px",
+  },
+  half: {
+    width: "450px",
+    height: "50px",
+    margin: "30px",
+  },
+  memo: {
+    width: "960px",
+    height: "50px",
+    border: "1px solid #9EADBA",
+    paddingLeft: "20px",
+    color: "black",
+  },
+});
+
 export const TransferGrain = ({ledger, setLedger}: Props) => {
-  const [sourceIdentity, setSourceIdentity] = useState<Identity | null>(null);
-  const [destIdentity, setDestIdentity] = useState<Identity | null>(null);
+  const classes = useStyles();
+  const [sourceAccount, setSourceAccount] = useState<Account | null>(null);
+  const [destAccount, setDestAccount] = useState<Account | null>(null);
   const [amount, setAmount] = useState<string>("0");
-  const [maxAmount, setMaxAmount] = useState<Grain>(fromInteger(0));
   const [memo, setMemo] = useState<string>("");
 
   const setSender = (acct: Account | null) => {
-    setMaxAmount(acct ? acct.balance : fromInteger(0));
-    setSourceIdentity(acct ? acct.identity : null);
+    setSourceAccount(acct);
   };
 
   const setReceiver = (acct: Account | null) => {
-    setDestIdentity(acct ? acct.identity : null);
+    setDestAccount(acct);
   };
```

Comment:
Now we should be able to remove these declarations entirely, and just set them as `const [sender, setSender] = useState<Account|null>(null)`. Since this assignment reduces to `const setSender = setSourceAccount`.